### ISSUE TITLE
TPCH - Why only queries 1-7?

### ISSUE THREAD

**Issue by mrocklin** [coiled/benchmarks#1071]
This is a genuine question, not a suggestion or a request for work.  Why did we choose to focus on these seven queries.  Are they special in some way?

@phofl I suspect that this question is mostly for you.

**Comment by phofl** [comment]
I agree

**Comment by mrocklin** [comment]
@milesgranger if you have time tomorrow can I ask you to bring query 8 over from Polars into the system we have here?

**Comment by phofl** [comment]
I think @mrocklin also wanted a Dask implementation? That's what I understood when we chatted yesterday

**Comment by mrocklin** [comment]
Yeah, ideally we'd have coverage for all of the projects supported here,
Dask, DuckDB, Polars, and Spark

**Comment by milesgranger** [comment]
> bring query 8 over from Polars

I took that too literally. :sweat_smile:
